packages/client/src/client/notificationsUtils.ts

```diff
@@ -40,3 +44,15 @@ export const processCreditQuote = (quote: PricedQuoteDetails) => {
     )} @ $${formatNumber(quote.state.payload)}`,
   }
 }
+
+export const processRFQRequestConfirmation = ({
+  request,
+  rfqId,
+}: ConfirmCreatedCreditRfq) => {
+  return {
+    title: `RFQ Created: RFQ ID ${rfqId}`,
+    rfqDetails: `You have sent a ${request.direction} RFQ for ${formatNumber(
+      request.quantity * 1000,
```

Comment:
I asked Tammy to provide some centralised utility to manage RFQ quantity but I don't think that was done
.. as it is easy to miss a case where we are not adjusting by a factor of 1000 - please add a utility, or at least a constant everywhere we are adjusting quantity, which is entered by user as integer (I think) multiple of 000s